src/internal/ActionStack.js

```diff
@@ -199,6 +199,11 @@ class ActionStack extends Smart {
     }
   }
 
+  getInstances(key) {
+    const groups = _.groupBy(this.stack, 'key');
+    return groups[key].map((item) => item.instance);
```

Comment:
Maybe just make sure that this key exists before returning, like : 
`return _.isArray(groups[key]) ? groups[key].map((item) => item.instance) : [];`